app/controllers/api/v1/rents_controller.rb

```diff
@@ -5,11 +5,12 @@ class RentsController < ApiController
 
       def create
         rent = Rent.create(rent_params)
+        Api::V1::RentsMailer.new_rent(rent).deliver_later
```

Comment:
This mail should be send only if the creation is a success. But you are implementing this in the previous PR ;)